app/presenters/queries/content_item_presenter.rb

```diff
@@ -223,11 +223,14 @@ def join_lateral_aggregates(scope)
       def parse_results(results)
         json_columns = %w(details routes redirects state_history unpublishing links)
         int_columns = %w(user_facing_version lock_version)
+        psql_array_column = "auth_bypass_ids"
 
         Enumerator.new do |yielder|
           results.each do |result|
             json_columns.each { |c| parse_json_column(result, c) }
             int_columns.each { |c| parse_int_column(result, c) }
+            parse_psql_array_column(result, psql_array_column)
```

Comment:
I think the `psql_array_column` variable is unnecessary, and would just pass `"auth_bypass_ids"` directly, as it's not being reused and it's already clear from the method name that we're passing the name of a column.

Reply:
Sure!